src/main/java/org/example/algorithms/Kruskal.java

```diff
@@ -0,0 +1,189 @@
+// Java program for Kruskal's algorithm to
+// find Minimum Spanning Tree of a given
+// connected, undirected and weighted graph
+
+import java.io.*;
+import java.lang.*;
+import java.util.*;
+
+class Graph {
```

Comment:
- Rename class to Kruskal
- Add package info at top of class.
- Follow naming conventions as mentioned in https://github.com/BimalRajGyawali/hacktoberfest-java/blob/main/CONTRIBUTING.md#naming-conventions